microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -38,6 +38,7 @@ class CloudHypervisorTests(Tool):
     ms_igvm_parser_repo = ""
     use_ms_clh_repo = False
     ms_access_token = ""
+    clh_guest_vm_type = ""
```

Comment:
We can set the default value to "NON-CVM"

Reply:
Sure